docs/source/about/advanced_usage.rst

```diff
@@ -1558,6 +1559,38 @@ keybindings
 
       amd_vbaq = enabled
 
+`amd_enforce_hrd <https://localhost:47990/config/#amd_enforce_hrd>`__
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+   Enable Hypothetical Reference Decoder (HRD) enforcement to help constrain the target bitrate.
+
+   .. note:: This option only applies when using amdvce `encoder`_.
+
+**Default**
+   ``enabled``
+
+**Example**
+   .. code-block:: text
+
+      amd_enforce_hrd = enabled
+
+`amd_filler_data <https://localhost:47990/config/#amd_filler_data>`__
```

Comment:
I'm not sure that there's any reason that filler data would ever be desirable for our use case. My understanding of these two options is: HRD basically acts as a strict maximum bitrate cap, while filler data acts as a strict minimum bitrate cap (though there are numerous issues with it not working properly in the AMF bug tracker).

Filler data is literally just appending filler NALUs with all zero data. It doesn't increase the usable bitrate in any way. I don't see the point in wasting time, power, and bandwidth to take those zero bytes and compute FEC, encrypt, and send them across the wire, then have to do the reverse again on the client side. And if we don't receive enough of the frame on the client side, we might have to throw away an entirely good frame just because we dropped some packets full of zeros on the wire.

Reply:
I suppose the most obvious reason why the option is available is due to HRD compliance. The stream is not supposed to underflow or overflow, and filler data prevents the former case from occuring, whilst `enforce_hrd` mostly (exclusively?) handles overflow. 

Underflow shouldn't be an issue unless we need to support a picky decoder that requires full HRD compliance, but I'm not aware of any such cases, so I assume that won't ever be an issue for us. I'll back out the filler data commit.

Reply:
Okay, sounds good to me. We could also ask the AMF folks to get a more authoritative answer if you want. They tend to be pretty responsive.